connexion/decorators/validation.py

```diff
@@ -19,10 +19,29 @@
 TYPE_MAP = {
     'integer': int,
     'number': float,
-    'boolean': boolean
+    'boolean': boolean,
+    'object': dict
 }
 
 
+class PropertyValidationError(Exception):
```

Comment:
I'm pretty sure jsonschema should handle nested objects and additionalProperties.
Is this really needed? What am I missing?